lib/ReactViews/Workbench/Controls/ViewingControls.tsx

```diff
@@ -269,9 +269,6 @@ class ViewingControls extends React.Component<
   }
 
   openDiffTool() {
-    // Disable timeline
-    // Should we do this? Difference is quite a specific use case
-    this.props.item.terria.timelineStack.removeAll();
```

Comment:
This was a dodgy behavior which resulted in losing timeline items when entering and leaving difftools. Instead we now simply hide/show the timeline when entering/leaving difftool (check https://github.com/TerriaJS/terriajs/blob/98cd1e3f9ab81d6ee57fb0f5d6ee2574d847f595/lib/ReactViews/Tools/DiffTool/DiffTool.tsx)